img/p/payroll.payworks.ca.svg

```diff
@@ -0,0 +1,10 @@
+    </pattern>
+  </defs>
+  <g id="Group_3246" data-name="Group 3246" transform="translate(-144 -17)">
+    <rect id="Payworks-4C" width="217" height="64" transform="translate(144 17)" fill="url(#pattern)"/>
+  </g>
+</svg>
```

Comment:
The SVG you've uploaded is just a PNG file wrapped in SVG code.
I suggest you use this image instead:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="url(#a)" viewBox="0 0 47.3 47.3"><defs><linearGradient id="a"><stop stop-color="#f05867"/><stop offset="1" stop-color="#f68c4c"/></linearGradient></defs><circle cx="33.38" r="18.98" stroke-width="3" transform="rotate(45)"/><circle cx="-33.38" r="22" stroke-width="3.1" transform="rotate(-135)"/></svg>
```